### PR TITLE
new-convo-styling

### LINKED ISSUE #511: nextjs: new convo page missing styling
image

### PR DESCRIPTION
**Pull request by McPizza0**
## What does this PR do?

adds styling to new convo

Fixes #511

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#518** <a href="https://app.graphite.dev/github/pr/un/inbox/518?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortCode]/convo/_components/create-convo-form.tsx [hunk 1]
```diff
@@ -290,7 +292,7 @@
     return false;
   }, [isTextPresent, topic, selectedParticipants, selectedEmailIdentity]);
 
-  async function startConvoUnderlying(type: 'message' | 'comment') {
+  async function startConvoCreation(type: 'message' | 'comment') {
     const getPublicIdsByType = (
       type: NewConvoParticipant['type'],
       property = 'publicId'
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortCode]/convo/_components/create-convo-form.tsx:296]: <i class='graphite__hidden'>[Re: line 301]<br/><br/></i>

@BlankParticle there is a type error on this line, can you check and fix please<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/un/inbox/518?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by BlankParticle — APPROVED** [review]
`manual_undefined` is kinda wired property value, We should name it something better like `no_avatar`. Else looks fine

**Comment by McPizza0** [comment]
### Merge activity

* **Jun 22, 3:15 AM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/518).

**Pull request merged**